cumulus_library/databases/duckdb.py

```diff
@@ -22,9 +22,14 @@
 class DuckDatabaseBackend(base.DatabaseBackend):
     """Database backend that uses local files via duckdb"""
 
-    def __init__(self, db_file: str):
+    def __init__(self, db_file: str, schema_name: str | None = None):
         super().__init__("main")
         self.db_type = "duckdb"
+        # As of the 1.0 duckdb release, local scopes, where schema names can be provided
+        # as configuration to duckdb.connect, are not supported.
+        # https://duckdb.org/docs/sql/statements/set.html#syntax
+
+        # This is where the connection config would be supplied when it is supported
```

Comment:
I'm electing to set up the schema stuff here, and in the DB utils call, even though the local module stuff is not configured yet, mostly just as breadcrumbs for the future.